conda/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = os.popen("python -c 'import versioneer; print(versioneer.get_version())'").read() %}
+
+package:
+  name: orion
+  version: {{ version }}
+
+source:
+  path: ../
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python {{ python }}
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - numpy
+    - scipy
+    - pyyaml
+    - pymongo >=3
```

Comment:
I think `gitpython` is missing here. When I install `orion` with conda I get 

```bash
pkg_resources.DistributionNotFound: The 'gitpython' distribution was not found and is required by orion.core
```
There's no way to automatically reuse the list from setup.py I guess? 

Also, can we add a test before the deployment to make sure such missing dependencies does not happen again in the future?

One final thing, can we list the package as orion.core to stay consistent with PiPy?

Reply:
No there is no way to sync with setup.py. 

I am testing that the produced package imports fine, but we might do some more tests to make sure it actually works.  I'm not sure which though.

Reply:
Just running `orion --help` after installation with conda would at least verify the dependencies.

Reply:
@abergeron Any updates on this? I guess you were very busy with the IVADO classes last week. Would you have time to fix the requirements issue and add a test this week?

Reply:
Maybe, but I have the git tutorial to give, so probably not until Friday.  If I find some time before that, I'll do it.

Otherwise, I don't mind if you want to do it to get it done faster.

Reply:
Oh, sure, I forgot about the tutorial. I'll reserve my time for #137 and #128 for now. Thanks for you help!